testbeam_analysis/tools/plot_utils.py

```diff
@@ -632,6 +826,58 @@ def plot_events(input_tracks_file, event_range, dut, max_chi2=None, output_pdf_f
     output_pdf_file : string
         Filename of the output PDF file. If None, the filename is derived from the input file.
     '''
+
+    if gui:
+        with tb.open_file(input_tracks_file, "r") as in_file_h5:
+            fitted_tracks = False
+            try:  # data has track candidates
+                table = in_file_h5.root.TrackCandidates
+            except tb.NoSuchNodeError:  # data has fitted tracks
+                table = in_file_h5.get_node(in_file_h5.root, name='Tracks_DUT_%d' % dut)
+                fitted_tracks = True
+
+            n_duts = sum(['charge' in col for col in table.dtype.names])
+            array = table[:]
+            tracks = testbeam_analysis.tools.analysis_utils.get_data_in_event_range(array, event_range[0], event_range[-1])
+            if tracks.shape[0] == 0:
```

Comment:
@DavidLP Wouldn't it make more sense to give an `n_events` parameter instead of an event range, since you never now from which event number on you have tracks to plot? e.g `event_range`=(0, 40) for the `fei4_telescope` data just returns without plotting since the first track in the table has `event_number`~80000. Instead just say plot the first `n_tracks`